javascripts/background_scripts/posting_process.js

```diff
@@ -242,17 +242,5 @@ chrome.runtime.onMessage.addListener(
     }
   });
 
-// Respond to the request sent from posting_button.js with the value from localStorage["Options:DissableButton"]
-chrome.runtime.onMessage.addListener(
-  function(request, sender, sendResponse) {
```

Comment:
hi, posting button is changed to directly listen to option changed events:
https://github.com/privly/privly-chrome/pull/124/files#diff-d38f979261233fa789d1599f4b6f4feaR124